export/tasks_generator_test.go

```diff
@@ -153,15 +153,6 @@ func makeAllDependencies(dd directDependencies) allDependencies {
 	return &allDependenciesSet{dd: dd, s: make(map[parade.TaskID]map[parade.TaskID]struct{})}
 }
 
-func validateAllAfter(t *testing.T, d allDependencies, typ string, before parade.TaskID, afters []parade.TaskID) {
```

Comment:
This one's mine.  Note to self: verify that we don't really need to use it any longer.

Reply:
Reply to self: D'uh, you removed it in favour of a better checked with full transitivity that lets you test dependencies directly, e.g. `expectedDeps` on l.358.